### PR TITLE
Add russian and ukrainian localizations.

### PR DESCRIPTION
**Pull request by volodymyrd**
## :scroll: Description
Add translations for Russian and Ukrainian languages

## :bulb: Motivation and Context
I'd like to have both languages in my project

## :green_heart: How did you test it?
locally, using the examle

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [X] All tests passing
- [X] No breaking changes

### CODE REVIEW

File: feedback/lib/src/l18n/translation.dart [hunk 1]
```diff
@@ -138,6 +174,8 @@ class GlobalFeedbackLocalizationsDelegate
     const Locale('de'): const DeFeedbackLocalizations(),
     const Locale('fr'): const FrFeedbackLocalizations(),
     const Locale('ar'): const ArFeedbackLocalizations(),
+    const Locale('ru'): const RuFeedbackLocalizations(),
+    const Locale('uk'): const UkFeedbackLocalizations(),
```
Comment (ueman) [inline feedback/lib/src/l18n/translation.dart:178]: @volodymyrd Do you mind double-checking this? Sometimes it's `uk` and sometimes it's `ua`. What is the more correct one?

**Review by ueman — COMMENTED** [review]
Thanks for doing this, I appreciate it. I have a small question, but otherwise this looks good to me.

File: feedback/lib/src/l18n/translation.dart (hunk 1 above)
Comment (volodymyrd) [reply feedback/lib/src/l18n/translation.dart:178]: yes, sure, please take a look at:
uk  - it's for the [language code](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=uk)
UA - it's for the [country code](https://en.wikipedia.org/wiki/ISO_3166-2:UA) 
full local is uk_UA, but your app uses only a language code, so uk is correct one

Comment (ueman) [reply feedback/lib/src/l18n/translation.dart:178]: I see, thanks for the clarification.

**Pull request merged**